### ISSUE TITLE
Adding a custom designed terrain in blender to LGSVL simulator.

### ISSUE THREAD

**Issue by siddharthjoshi01** [lgsvl/simulator#1078]
I have designed a custom terrain using blender 2.9, and exported the same to .fbx format in Unity. My objective is to get this terrain into the LGSVL simulator and test the terrain over a vehicle of my choice. I am working on a windows machine, and not able to find any guidelines. Need help.

**Comment by EricBoiseLGSVL** [comment]
https://www.lgsvlsimulator.com/docs/add-new-map/
Let us know if you have issues ;)

**Comment by siddharthjoshi01** [comment]
Yes, I have been through this video. However, it involves getting a ready-to-use downloadable asset-bundle from the asset store. In my case, I have a custom terrain designed as attached below. How to get it into LGSVL after exporting as .fbx? Will I have to create an asset-bundle myself?
![blender_terrain](https://user-images.githubusercontent.com/57046434/96629128-087b7580-12c8-11eb-8cb0-306ccb19a5d2.PNG)

**Comment by EricBoiseLGSVL** [comment]
So just replace importing the asset store assets to right click in Project panel in the folder for your new map and Import New Asset...
![ImportHelp00](https://user-images.githubusercontent.com/45106230/96634524-a3844780-12e8-11eb-96d1-ce1932358377.png)
